diff --git a/src/libprojectM/MilkdropPreset/IdlePreset.cpp b/src/libprojectM/MilkdropPreset/IdlePreset.cpp
index 747d4d3f5..d5c8b2fad 100644
--- a/src/libprojectM/MilkdropPreset/IdlePreset.cpp
+++ b/src/libprojectM/MilkdropPreset/IdlePreset.cpp
@@ -186,7 +186,7 @@ per_frame_29=q3 = if(above(frame,60),1, frame/60.0);
 per_frame_30=oldq8 = if(above(oldq8,1000),0,oldq8);
 per_frame_31=cx = cx + 0.3 * (0.6 * sin(0.245 * time) + 0.4 * sin(0.123 * time));
 per_frame_31=cy = cy + 0.3 * (0.6 * sin(0.263 * time) + 0.4 * sin(0.117 * time));
-per_frame_32=q9 = -0.12; // Headphones Y offset
+per_frame_32=q9 = 0.12; // Headphones Y offset
 per_frame_33=q10 = 0.5 * sin(time * 0.5623); // Logo tilt
 per_pixel_1=zoom = log(sqrt(sin(time * .5) * 0.5 + 2.5) - .5 * rad) + 0.4;
 )";
